dotcom-rendering/src/components/CardHeadline.tsx

```diff
@@ -166,6 +166,9 @@ const sublinkStyles = css`
 	font-size: inherit;
 	font-weight: ${fontWeights.medium};
 	line-height: inherit;
+	${until.tablet} {
+		min-height: 44px;
+	}
```

Comment:
Should we rather use the [`pointer: coarse` media query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer)?

---

What about using `space[12]` from Source?

Reply:
I want 44px not 48px